.github/workflows/build.yml

```diff
@@ -8,6 +8,13 @@ on:
   push:
     branches:
       - master
+    - '**'
```

Comment:
Why did you change this, what's the intention?

Reply:
And it causes an error:
```
Invalid workflow file: .github/workflows/build.yml#L8
You have an error in your yaml syntax on line 8
```

Reply:
So it won't run when other yml files are changed. Trivial in this case, but I figured you might want to add more path filters in the future:

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Reply:
I don't see why, at least for this case it seems a bit beyond the goal.

If we have use cases for excluding files from triggering a CI run, we
can add them but no excluding only two YAML files doesn't change much.

Without having checked it, I think about 95% of the commits will change files which should trigger the CI anyway, so don't waste resources maintaining an articial exclude list.

Reply:
I totally see your point. I've reverted all changes to build.yml.